wrappers/internal/service_unit_gen.go

```diff
@@ -282,10 +294,19 @@ WantedBy={{.ServicesTarget}}
 
 		InterfaceServiceSnippets: ifaceSpecifiedServiceSnippet,
 		InterfaceUnitSnippets:    ifaceSpecifiedUnitSnippet,
+		Restart:                  restartCond,
+
+		// When converting a Duration to a string, Golang produces units "ns",
+		// "µs", "ms", "s", "m", and "h". These are understood by systemd (see
```

Comment:
it seems that µs is only accepted after this commit https://github.com/systemd/systemd/commit/d0a6d7c4d1285d6af2be299a466c964f0c5991a2 which landed in systemd 254

looking at the Go side: https://go.dev/play/p/6VD6xYd-Q52 it defaults to `µs` pretty consistently (I think this may always have been the case), so we're likely already producing invalid units on Jammy and earlier